commit 2ea93838fcd6ef6e8c9325e964b1a04e6ac6985b
Author: Eric Anholt <eric@anholt.net>
Date:   Wed Aug 2 11:46:51 2017 -0700

    drm/vc4: Don't disable DSI clocks on component unload.
    
    The clocks are enabled/disabled at encoder enable/disable time, not at
    component load.  Fixes a WARN_ON at boot if V3D fails to probe.
    
    Fixes: 4078f5757144 ("drm/vc4: Add DSI driver")
    Signed-off-by: Eric Anholt <eric@anholt.net>

diff --git a/drivers/gpu/drm/vc4/vc4_dsi.c b/drivers/gpu/drm/vc4/vc4_dsi.c
index 17fcac381dbb..284f486ed9db 100644
--- a/drivers/gpu/drm/vc4/vc4_dsi.c
+++ b/drivers/gpu/drm/vc4/vc4_dsi.c
@@ -1692,9 +1692,6 @@ static void vc4_dsi_unbind(struct device *dev, struct device *master,
 
 	mipi_dsi_host_unregister(&dsi->dsi_host);
 
-	clk_disable_unprepare(dsi->pll_phy_clock);
-	clk_disable_unprepare(dsi->escape_clock);
-
 	if (dsi->port == 1)
 		vc4->dsi1 = NULL;
 }
